lesson6/Task4/B.java

```diff
@@ -0,0 +1,20 @@
+package lesson6.Task4;
+
+public class B extends A{
+    B (String nata, int name) {
+    super(nata, name);
+    };
+    @Override
+    void lesson2() {
+        System.out.println("second method from class B");
+        System.out.println("second method from class B");
```

Comment:
"Override another method to add some extra functionality to it. "
So, you need to invoke super method here and add something extra.